example/Tests/worklet-tests.ts

```diff
@@ -197,6 +197,35 @@ export const worklet_tests = {
     });
     return ExpectValue(result, 42);
   },
+  check_jsi_object_is_spreadable_after_worklet: async () => {
+    const fw = () => {
+      "worklet";
+      return { a: 100 };
+    };
+    let wf = Worklets.defaultContext.createRunAsync(fw);
+    const result = await wf();
+    const spreadObject = { ...result };
+    return ExpectValue(spreadObject, { a: 100 });
+  },
+  check_jsi_object_is_assignable_after_worklet: async () => {
+    const fw = () => {
+      "worklet";
+      return { a: 100 };
+    };
+    let wf = Worklets.defaultContext.createRunAsync(fw);
+    const result = await wf();
+    const assignedObject = Object.assign({}, result);
```

Comment:
I thought the problem was when using Object.assign/spread **inside a Worklet**?

Reply:
No, I have added a test below to show that spreading inside the worklet works as expected.
My use case was to spread the object that is returned by the worklet, and I have not spread inside worklet in production code, so I have no further experience with it.